packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -33,7 +33,11 @@ export class Popup extends React.Component<IPopupProps, IPopupState> {
 
   public componentDidMount(): void {
     if (this._root.current) {
-      this._disposables.push(on(this._root.current, 'focus', this._onFocus, true), on(this._root.current, 'blur', this._onBlur, true));
+      this._disposables.push(
+        on(this._root.current, 'focus', this._onFocus, true),
+        on(this._root.current, 'blur', this._onBlur, true),
+        on(getWindow(this._root.current) || window, 'keydown', this._onKeyDown as any)
```

Comment:
don't think you need `|| window`

Reply:
If you look at `getWindow` it will do that by default.

Reply:
without || window the on() typings aren't happy. getWindow can return undefined, which on doesn't like.